### PR TITLE
Modified the metrics endpoint to return the top rated Staff Picks

### LINKED ISSUE #1077: Modify the metrics endpoint to return the top rated Staff Picks
**Actual Behaviour**

Currently, such a feature is not there.

**Expected Behaviour**

This needs to be implemented to facilitate displaying the Staff Picks on the CMS Homepage.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #1077 

Changes: Modified the metrics endpoint to return the top rated Staff Picks. The sample JSON response can be seen [here](https://jsoneditoronline.org/?id=59e0da477d0c48ef99789b3b4be29d82).

### CODE REVIEW

File: src/ai/susi/server/api/cms/SkillMetricsDataService.java [hunk 1]
```diff
@@ -112,6 +113,10 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
 
                     jsonArray.put(skillMetadata);
                     skillObject.put(skill_name, skillMetadata);
+
+                    if(SusiSkill.isStaffPick(model_name, temp_group_name, language_name, skill_name)) {
```
Comment (akshatnitd) [inline src/ai/susi/server/api/cms/SkillMetricsDataService.java]: This block of code also needs to be written when group name is not `All`

Comment (akshatnitd) [reply src/ai/susi/server/api/cms/SkillMetricsDataService.java]: Please add the lines.

Comment (Akshat-Jain) [reply src/ai/susi/server/api/cms/SkillMetricsDataService.java]: Done. Please review. Thanks :-)

**Comment by Akshat-Jain** [comment]
@DNS-404 @akshatnitd Resolved the conflicts. Please review. Thanks :-)

**Pull request merged**